.github/workflows/sdk-release.yml

```diff
@@ -93,7 +93,7 @@ jobs:
       fail-fast: false
       max-parallel: 1
       matrix:
-        repo: [block-explorer, 0nft, 0box, 0dns, 0proxy, zwalletcli, zboxcli, zboxmobile, blobber ]
+        repo: [block-explorer, 0nft, 0box, 0dns, 0proxy, zwalletcli, zboxcli, zboxmobile, blobber, s3server, s3-migration ]
```

Comment:
block-explorer, 0proxy and zboxmobile can be removed now.

Reply:
s3server should be `zs3server`. and please add https://github.com/0chain/zboxcli/blob/staging/.github/workflows/gosdk.yml on zs3server and s3-migration repos.

Reply:
fixed